message_sender/tests.py

```diff
@@ -3175,6 +3175,79 @@ def test_requeue(self):
         self.assertEqual(OutboundSendFailure.objects.all().count(), 0)
 
 
+class TestFailedMsisdnLookUp(TestCase):
+    def add_identity_no_address_search_response(self, msisdn, identity, count=0):
+        response = {"next": None, "previous": None, "results": []}
+        responses.add(
+            responses.GET,
+            "%s/identities/%s/addresses/msisdn"
+            % (settings.IDENTITY_STORE_URL, identity),  # noqa
+            json=response,
+            status=200,
+        )
+
+    def add_metrics_response(self):
+        responses.add(
+            responses.POST, "http://metrics-url/metrics/", json={}, status=201
+        )
```

Comment:
I don't think that we need to mock out this metrics response, since we're not getting to the end of the task where the metric is being sent.

Reply:
I get a metric error without it, do I still need to take it out?

Reply:
actually, you are correct I don't need the metric response